tests/testthat/test-json.R

```diff
@@ -81,7 +81,7 @@ test_that("JSON parser, real examples", {
       "email": "h.wickham@gmail.com",
       "date": "2015-03-30T13:55:18Z"
     },
-    "message": "Merge pull request #22 from paulstaab/master\\n\\nImprove error message for assertions of length 0",
+    "message": "Merge pull request #22 from paulstaab/HEAD\\n\\nImprove error message for assertions of length 0",
```

Comment:
You cannot change this I believe, or the tests will fail.

Reply:
This is just testing that the JSON parser works and the test is passing.